File: webui/src/app/models/fixture.ts
export interface Fixture {
  id: number;
  referee?: string;
  date: string;
  venueId: number;
  league: string;
  season?: string;
  round?: string;
  homeTeamId: number;
  logoHome: string;
  awayTeamId: number;
  logoAway: string;
  goalsHome?: number;
  goalsAway?: number;
  PenaltyHome?: number;
  PenaltyAway?: number;
  rate?: number;
  userNote?: number;
}

File: webapi2/FutRank/FutRank/Models/Fixture.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace FutRank.Models
{
    public class Fixture
    {
        [Key]
        public int Id { get; set; }
        public string? Referee { get; set; }
        public string Date { get; set; }
        public int VenueId { get; set; }
        public int LeagueId { get; set; }
        public string? Season { get; set; }
        public string? Round {  get; set; }
        public int HomeTeamId {  get; set; }
        public int AwayTeamId { get; set; }
        public int? GoalsHome { get; set; }
        public int? GoalsAway { get; set; }
        public int? PenaltyHome { get; set; }
        public int? PenaltyAway { get; set; }
        public float? Rate { get; set; }
        public int Time { get; set; }

        public League League { get; set; }
        public Venue Venue { get; set; }
        public Club HomeClub { get; set; }
        public Club AwayClub { get; set; }

        public virtual ICollection<UserFixtures> UserFixtures { get; set; }
        public ICollection<CommentFixture> Comments { get; set; }
    }
}
